kube_resource_report/report.py

```diff
@@ -97,6 +97,21 @@ def query_cluster(
     logger.info(f"Querying cluster {cluster.id} ({cluster.api_server_url})..")
     pods = {}
     nodes = {}
+    namespaces ={}
```

Comment:
a little explanation here:

this is a preparation for a later PR. we annotate the namespace with the email of the responsible. with this information it will be possible to send out emails (generated by a cron job) to the responsible about the resource usage in the corresponding namespace.